build.sh

```diff
@@ -12,11 +12,15 @@ get_yaml_string() {
     yq -- "$1" "$RECIPE_FILE"
 }
 
+# Read configuration variables.
+fedora_version="$(get_yaml_string '.fedora-version')"
+
 # Add custom repos.
 get_yaml_array repos '.extrarepos[]'
 if [[ ${#repos[@]} -gt 0 ]]; then
     echo "-- Adding repos defined in recipe.yml --"
     for repo in "${repos[@]}"; do
+        repo="${repo//%FEDORA_VERSION%/${fedora_version}}"
```

Comment:
**Breaking edge case**
If `fedora-version: latest` (supported configuration), this doesn't work.
Possible solution: pass an `ABSOLUTE_FEDORA_VERSION` to the Containerfile from `build.yml`, where the version is queried from the base image somehow ([see line 88?](https://github.com/ublue-os/startingpoint/blob/cd5c6c33cbb6cbcd5b75cb4be8fb7d10e4b59140/.github/workflows/build.yml#LL88C16-L88C16)). Or if there's some way to get the fedora version using bash scripting, that'd be better.

Can't merge before fix. Be this multiple separate pull requests, this wouldn't delay other fixes.

Reply:
@EinoHR Nice catch. If you're thinking of passing a variable from the Containerfile, I would love to be able to DELETE `fedora-version` from `recipe.yml` by the way, so that it's only specified in Containerfile for ultimate maintainability. :)